aio_pika/robust_connection.py

```diff
@@ -89,8 +104,7 @@ def close(self) -> None:
         self._closed = True
 
         while self._on_close_callbacks:
-            with suppress(Exception):
```

Comment:
@mosquito I guess any exception should be caught within callbacks for explicit.

Reply:
You have to add `finally` block for guarantee to call close.

Reply:
@mosquito Yep.